### PR TITLE
chore(release): 1.7.1 [skip ci]

### PR DESCRIPTION
**Pull request by lucasmcdonald3**
## Note: Semantic release wants to cut this as 1.8.0 because the Python MPL was merged as a `feat`. I manually changed this to a 1.7.1 release.

---

# [1.7.1](https://github.com/aws/aws-cryptographic-material-providers-library/compare/v1.7.0...v1.7.1) (2024-10-11)

This release is available in the following languages:

- Python

### Bug Fixes

* **H-Keyring:** if getCache returns Error not EntryDoesNotExist, error ([#846](https://github.com/aws/aws-cryptographic-material-providers-library/issues/846)) ([3413fcb](https://github.com/aws/aws-cryptographic-material-providers-library/commit/3413fcbc889e2c31fb753ff3eca852c1d5db1140))
* H-Keyring: if putCache throws EntryAlreadyExists, swallow ([#856](https://github.com/aws/aws-cryptographic-material-providers-library/issues/856)) ([d01a182](https://github.com/aws/aws-cryptographic-material-providers-library/commit/d01a182b5e62010f44616dacd6730aa873dec4c7))
* **Python:** Repolymorph to add orphaned shapes ([#831](https://github.com/aws/aws-cryptographic-material-providers-library/issues/831)) ([2481dd8](https://github.com/aws/aws-cryptographic-material-providers-library/commit/2481dd8dbad94213acbec25441421ff6d1eb104e))

### Features

* Python MPL ([#805](https://github.com/aws/aws-cryptographic-material-providers-library/issues/805)) ([cfb2f7e](https://github.com/aws/aws-cryptographic-material-providers-library/commit/cfb2f7e06c406b3f3ddb506a6356382737b53746))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by texastony — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+This release is available in the following languages:
+
+- Python
+
+This is the first release for the Python implementation of the AWS Cryptographic Material Providers Library. ([#805](https://github.com/aws/aws-cryptographic-material-providers-library/issues/805)) ([cfb2f7e](https://github.com/aws/aws-cryptographic-material-providers-library/commit/cfb2f7e06c406b3f3ddb506a6356382737b53746))
+
+### Bug Fixes
+
+- **H-Keyring:** if getCache returns Error not EntryDoesNotExist, error ([#846](https://github.com/aws/aws-cryptographic-material-providers-library/issues/846)) ([3413fcb](https://github.com/aws/aws-cryptographic-material-providers-library/commit/3413fcbc889e2c31fb753ff3eca852c1d5db1140))
```
Comment (texastony) [inline CHANGELOG.md:13]:
```suggestion
- **H-Keyring:** if getCache returns Error not EntryDoesNotExist, raise error ([#846](https://github.com/aws/aws-cryptographic-material-providers-library/issues/846)) ([3413fcb](https://github.com/aws/aws-cryptographic-material-providers-library/commit/3413fcbc889e2c31fb753ff3eca852c1d5db1140))
```

**Review by texastony — APPROVED** [review]
Tiny nit.

**Review by ajewellamz — DISMISSED** [review]

**Review by josecorella — DISMISSED** [review]

File: AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts [hunk 2]
```diff
@@ -18,7 +18,7 @@ var props = Properties().apply {
 var dafnyVersion = props.getProperty("dafnyVersion")
 
 group = "software.amazon.cryptography"
-version = "1.7.0-SNAPSHOT"
+version = "1.7.1"
```
Comment (lucasmcdonald3) [inline AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:21]:
```suggestion
version = "1.7.1" 
```

File: AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts [hunk 3]
```diff
@@ -18,7 +18,7 @@ var props = Properties().apply {
...
```
Comment (lucasmcdonald3) [inline AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts:21]:
```suggestion
version = "1.7.1"
```

**Review by ajewellamz — DISMISSED** [review]

File: AwsCryptographicMaterialProviders/runtimes/net/AssemblyInfo.cs [hunk 4]
```diff
@@ -3,4 +3,4 @@
 [assembly: AssemblyTitle("AWS.Cryptography.MaterialProviders")]
 
 // This should be kept in sync with the version number in MPL.csproj
-[assembly: AssemblyVersion("1.7.0")]
+[assembly: AssemblyVersion("1.7.1")]
```
Comment (lucasmcdonald3) [inline AwsCryptographicMaterialProviders/runtimes/net/AssemblyInfo.cs:6]:
```suggestion
[assembly: AssemblyVersion("1.7.1")] 
```

**Review by ajewellamz — APPROVED** [review]

File: AwsCryptographicMaterialProviders/runtimes/net/AssemblyInfo.cs [hunk 5]
```diff
@@ -3,4 +3,4 @@
...
```
Comment (lucasmcdonald3) [inline AwsCryptographicMaterialProviders/runtimes/net/AssemblyInfo.cs:6]:
```suggestion
[assembly: AssemblyVersion("1.7.1")]
```

**Review by josecorella — APPROVED** [review]

**Pull request merged**